### PR TITLE
[ADVAPP-531] Introduce Ability to Add a Journey Step to Invite a Student or Prospect to an Event

### PR DESCRIPTION
**Pull request by ankit-canyon**
### Ticket(s) or GitHub Issue

- https://canyongbs.atlassian.net/browse/ADVAPP-531

### Technical Description

> Describe your changes in detail. This should include technical/architectural decisions and reasoning, if applicable.
> Introduce Ability to Add a Journey Step to Invite a Student or Prospect to an Event

_______________________________________________

### CODE REVIEW

File: app-modules/meeting-center/src/Models/Event.php [hunk 1]
```diff
@@ -71,4 +76,39 @@ public function attendees(): HasMany
     {
         return $this->hasMany(EventAttendee::class, 'event_id');
     }
+
+    public static function executeFromCampaignAction(CampaignAction $action): bool|string
+    {
+        if (app(LicenseSettings::class)->data->addons->eventManagement) {
```
Comment (Orrison) [inline app-modules/meeting-center/src/Models/Event.php:82]: I think this is the first time that we have a campaign action that is gated by a feature/addon.

What happens if this is attempted to be executed when the addon is turned off? Does it retry to run indefinitely?

**Review by Orrison — COMMENTED** [review]
I just pushed up one small change that I think might fix the random test failure.

Also I left one small comment to explore

**Review by Orrison — DISMISSED** [review]
Looks good! @ankit-canyon just let me know once you are done with local testing and we can merge this in.

**Review by Orrison — DISMISSED** [review]

**Comment by ankit-canyon** [comment]
> Looks good! @ankit-canyon just let me know once you are done with local testing and we can merge this in.

Yep All looks good ! everything is working smoothly.

**Review by Orrison — APPROVED** [review]

**Pull request merged**